### ISSUE TITLE
integrate with get-twitter-info

### ISSUE THREAD

**Issue by iamstarkov** [andrepolischuk/twitter-profile-media#3]
will look like this:

```js
import info from 'get-twitter-info';

export default (tokens, name, fn) => {
  info(tokens, name, (err, info) => {
    if (err) return fn(err);
    const image = res.profile_image_url_https.replace(/_normal/, '');
    const banner = res.profile_banner_url;
    fn(err, { image, banner });
  });
};
```

what do you think?

**Comment by andrepolischuk** [comment]
Good idea =) Thanks

**Comment by andrepolischuk** [comment]
Published in 1.0.1

**Comment by iamstarkov** [comment]
awsm
